src/sfa.Tl.Marketing.Communication/SearchPipeline/Steps/GetQualificationsStep.cs

```diff
@@ -28,9 +28,14 @@ public Task Execute(ISearchContext context)
             context.ViewModel.SelectedQualificationId = 0;
         }
 
+        const int HairdressingBarberingAndBeautyTherapyId = 53;
```

Comment:
This is really rubbish, but we believe data is handled by another service so don't want to remove from the database, just hiding